server.js

```diff
@@ -133,7 +139,8 @@ app.get('/download/:id', asyncHandler(async (req, res)=>{
 		'Content-Type'        : 'text/plain',
 		'Content-Disposition' : `attachment; filename="${fileName}.txt"`
 	});
-	res.status(200).send(brew.text);
+	const text = brew.text;
+	res.status(200).send(text);
```

Comment:
```suggestion
	res.status(200).send(brew.text);
```

We don't need to declare a new variable here since we don't do any processing on it before sending it out.